python/phylanx/ast/transformation.py

```diff
@@ -516,6 +519,36 @@ def convert_to_phylanx_type(v):
     return v
 
 
+class InitHPX:
+    cfg = \
+    [
+    # make sure hpx_main is always executed
+    "hpx.run_hpx_main!=1",
+    # allow for unknown command line options
+    "hpx.commandline.allow_unknown!=1",
+    # disable HPX' short options
+    "hpx.commandline.aliasing!=0",
+    # run one thread only (for now)
+    "hpx.os_threads!=1",
+    # don't print diagnostics during forced terminate
+    "hpx.diagnostics_on_terminate!=0",
+    # disable the TCP parcelport
+    "hpx.parcel.tcp.enable!=0"
+    ]
+    hpx_init = False
+
+    def __init__(self, config=cfg):
```

Comment:
I think that most of the settings above are mandatory for the functioning of HPX in the context of the Phylanx python frontend. The configuration settings passed to HPX should be the result of merging the user settings with the ones provided as a default.

Reply:
So I see that this sets the flags at runtime instead of compile time, but if this class is initialized in transformation.py, how does the programmer change the settings?